### PR TITLE
feat: fix cli issues

### PR DESCRIPTION
**Pull request by christyjacob4**
Fix for following issues:

1. Added selfSign as a user preference
2. Fix sh: unknown operand 
3. Rename Services folder to services

### CODE REVIEW

File: templates/cli/README.md.twig [hunk 1]
```diff
@@ -43,21 +43,17 @@ $ {{ language.params.executableName }} init
...
-X-{{ spec.title | caseUcfirst }}-Locale=en-US
+$ {{ language.params.executableName }} init --endpoint="YOUR ENDPOINT" --project="YOUR PROJECT ID" --key="YOUR API KEY" --locale="YOUR LOCALE"
 ```
 
+If any of the required values are missing, a prompt will show up, asking for those missing values.
+
 ### 2. Using the prompt
 
-Invoking `{{ language.params.executableName }} init` without an env file triggers an interactive prompt where you can manually enter the details. These values are saved locally so that you dont have to re-enter them everytime. 
+Invoking `{{ language.params.executableName }} init` without an params triggers an interactive prompt where you can manually enter the details. These values are saved locally so that you dont have to re-enter them everytime.
```
Comment (eldadfux) [inline templates/cli/README.md.twig:56]: `without an params triggers an interactive prompt where you can manually enter the details` fix grammar

**Review by eldadfux — CHANGES_REQUESTED** [review]

File: templates/cli/README.md.twig (hunk 1 above)
Comment (christyjacob4) [reply templates/cli/README.md.twig:56]: Good catch 😄

**Comment by eldadfux** [comment]
@christyjacob4 please don't forget to add the `setSelfSigned` as an example here: 

image

**Comment by christyjacob4** [comment]
@eldadfux  I have updated the docs

**Review by eldadfux — APPROVED** [review]

**Pull request merged**